src/settingsdialogplugins.cpp

```diff
@@ -164,6 +299,11 @@ void PluginsSettingsTab::on_pluginsList_currentItemChanged(QTreeWidgetItem *curr
   ui->versionLabel->setText(plugin->version().canonicalString());
   ui->descriptionLabel->setText(plugin->description());
 
+  ui->enabledCheckbox->setVisible(
```

Comment:
Why don't game plugins support disabling?

Reply:
Okay, "enabled" sounds more like a combination of "enabled" and "active". A plugin can be disabled because of the switch, because its master is disabled, etc., but also because it's a game plugin and it's not the managed game. I was expecting "enabled" to mean "this plugin can be loaded" along with a separate "this plugin is active" flag.